src/paris/mod.rs

```diff
@@ -0,0 +1,39 @@
+#[allow(missing_docs)]
```

Comment:
Can you maybe just steal the documentation from the [log crate](https://docs.rs/log/0.4.14/log/macro.info.html) and additionally add a link to [paris::formatter::colorize_string](https://docs.rs/paris/1.5.7/paris/formatter/fn.colorize_string.html) so people know where to look for more information?